odoo/tests/common.py

```diff
@@ -587,6 +587,31 @@ def assertTreesEqual(self, n1, n2, msg=None):
             self.assertTreesEqual(c1, c2, msg)
 
 
+    @contextmanager
+    def capture_trigger(self, cron_id=None):
```

Comment:
This is pretty specific, and two cases do not justify to put it on all tests' base class.
What about a mixin class inside `odoo/addons/base/tests/test_ir_cron.py`?